extensions/it-s-that-time/assets/scripts/large-message.py

```diff
@@ -0,0 +1,74 @@
+import sys
+import pyautogui
+from PyQt5.QtWidgets import QApplication, QLabel, QMainWindow, QVBoxLayout, QWidget, QSizePolicy
+from PyQt5.QtCore import Qt
+from screeninfo import get_monitors
```

Comment:
It looks like your extension requires some prerequisites before using.

These should be documented or solved with some automation scripts.

Reply:
Thanks !
I’ve added the prerequisites to the README